src/services/db.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('db', 'user', 'pass', {
```

Comment:
better to create constants
```suggestion
const DATABASE = 'db';
const USERNAME = 'user';
const PASSWORD = 'pass';

export const sequelize = new Sequelize(DATABASE, USERNAME, PASSWORD, {
```